tests/sentry/api/endpoints/test_organization_releases.py

```diff
@@ -940,16 +948,16 @@ def test_simple(self, mock_fetch_commits):
             {
                 'repository': 'test/repo',
                 'previousCommit': None,
-                'commit': 'previous-commit-id..current-commit-id',
+                'commit': 'previous-commit-id%scurrent-commit-id' % COMMIT_RANGE_DELIMITER,
```

Comment:
I switched to using the constant because we were already doing it that way here:

https://github.com/getsentry/sentry/blob/75ab9e5e3f254dfdffb133096eeb158b525499d9/src/sentry/models/release.py#L287-L288

but for these tests and the ones below, I feel like it makes them less readable. Thoughts?

Reply:
I think the literal value is fine in the test code as it makes the tests more readable.

Reply:
Done